tests/integration/test_backup_restore_on_cluster/test_disallow_concurrency.py

```diff
@@ -9,13 +9,13 @@
 
 cluster = ClickHouseCluster(__file__)
 
-num_nodes = 10
+num_nodes = 2
 
 
 def generate_cluster_def():
     path = os.path.join(
         os.path.dirname(os.path.realpath(__file__)),
-        "./_gen/cluster_for_concurrency_test.xml",
+        "./_gen/cluster_for_test_disallow_concurrency.xml",
```

Comment:
File "./_gen/cluster_for_concurrency_test.xml" could be used by different test `test_backup_restore_on_cluster/test_concurrency`. It's safer to use a different file name.